### PR TITLE
making some updates like dates and courseplus

### PR DESCRIPTION
**Pull request by carriewright11**
- updating dates
- courseplus
- made a new version of the survey for the summer

Just need slack workspace and might need to update grades due and assignment due date - I still haven't heard back.

Also need to decide what version of R to use... kinda between releases:

[R version 4.4.1 (Race for Your Life) prerelease versions](http://cran.r-project.org/src/base-prerelease) will appear starting Tuesday 2024-06-04. Final release is scheduled for Friday 2024-06-14.
[R version 4.4.0 (Puppy Cup)](https://cran.r-project.org/src/base/R-4) has been released on 2024-04-24.

### CODE REVIEW

**Review by avahoffman — APPROVED** [review]

**Comment by avahoffman** [comment]
Maybe we can just say >=4.4.0 for the version?

File: config.yml [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-  final_due_date: "January 24th at 11:59pm ET"
-  grades_due: "January 26"
-  holiday: "In recognition of [Martin Luther King Jr. Day](https://en.wikipedia.org/wiki/Martin_Luther_King_Jr._Day), there will be **no class on Monday January 15th, 2024**."
-  courseplus_web: "https://courseplus.jhu.edu/core/index.cfm/go/syl:syl.public.view/coid/20749/"
-  courseplus_dropboxes: "https://courseplus.jhu.edu/core/index.cfm/go/db:main/coID/20749/"
-  google_survey: "https://forms.gle/rTqJ4qwV2boo3FL78"
+  final_due_date: "June 28th at 11:59pm ET"
+  grades_due: "June 28th"
+  holiday: "In recognition of [Juneteenth](https://en.wikipedia.org/wiki/Juneteenth), there will be no class on Wednesday June 19th, 2023.**."
+  courseplus_web: "https://courseplus.jhu.edu/core/index.cfm/go/course.home/coid/21696/"
```
[the remark this answers is not in the document]
Comment (carriewright11) [reply config.yml:8]: sure that works for me

File: config.yml [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
+  courseplus_dropboxes: "https://courseplus.jhu.edu/core/index.cfm/go/db:admin.main/coid/21696/"
```
[the remark this answers is not in the document]
Comment (carriewright11) [reply config.yml]: done!

**Comment by carriewright11** [comment]
URL checker is just down right now so I am going to merge this

**Pull request merged**